### DISCUSSION TITLE
[PROPOSAL] Multiple Bumpkins

### DISCUSSION THREAD

**Discussion by c-py** [sunflower-land/sunflower-land#2211] [Ideas]
# Multiple Bumpkins 🧑‍🌾

_This proposal is a live document. Changes may be made based on player feedback or based on changing requirements_

Please feel free to place your feedback on the ideas below or if you have any ideas of your own you think would be fun in Sunflower Land!

## Introduction

Multiple Bumpkins introduces the ability for players to strategically use their Bumpkins Wearable Items and Bumpkin Skills. 

### Tent

The tent is an existing building. Keeping in line with the existing game design of the tent, placing a tent will allow a player to use a second Bumpkin on your farm. An additional tent will enable players to use a third Bumpkin on their farm.

![Tent](https://user-images.githubusercontent.com/41215134/217723147-4fb3c053-f8bc-414e-8e5b-f4ac96ce6777.png) 

The first Tent is currently available when a Bumpkin reaches level 7. The third tent would become available significantly later in the game which will require some balancing before a level can be published (TBD).

### Bumpkin Roles

_Based on player feedback, roles have been removed from the Multiple Bumpkins proposal. The new design will allow wearables from multiple bumpkins to be used at one time_

<s>The skill tree currently allows Bumpkins to specialise into the industries in Sunflower Land. Some players choose to specialise into crops first, while other's may priortise wood or mining. A Bumpkin's items allow the player to further specialise into a particular industry. Bumpkin roles offer the ability for players to strategically select which bumpkin they desire to perform a particular task. 

The idea behind Bumpkin Roles is allow specialist Bumpkins to perform specialist tasks. Wearable Items and Skills would apply for the resources within the Bumpkin role. For those who prefer to keep one Bumpkin performing all tasks, the role assignment allows for that as well.

![Bumpkin Roles](https://user-images.githubusercontent.com/41215134/217726286-a4588a66-d37d-4bbd-af5a-e3eac0f8e07a.png)

The roles mirror the skill tree:

- Crops
- Trees
- Rocks
- Cooking
- Animals</s>

### Skill Reset

Naturally, having the ability to specialise Bumpkins, requires the ability to respecialise Bumpkins. As more skill points become available, or when different industries are released, it may be advantageous for a player to reset one or more of Bumpkin's skills.

![Respec](https://user-images.githubusercontent.com/41215134/217727745-83833f4b-ece2-41f0-9bd7-264da1ecd943.png)
 
 Respecialising will come with a cost, increasing each time to a cap, resetting after one month. Values TBD.
 
 ## Final Note
 
 Please add any feedback on this proposal that you think is suitable.
 
 Thank you!

**Comment by brian-man** [comment]
Interesting.  Didn't expect to be able to leverage skills of multiple bumpkins.

How does eating work when you have multiple bumpkins (say 1 tent).  Do you choose at eat time which bumpkin is eating?  Similar question for expansions.

**Reply by c-py** [reply]
> How does eating work when you have multiple bumpkins (say 1 tent). Do you choose at eat time which bumpkin is eating?

Yes, the bumpkin living in the tent would appear on the farm, in the same manner as the bumpkin standing next to the mailbox on the farm. Potentially the Bumpkin could be place-able as well, although this hasn't been captured in the above proposal.

> Similar question for expansions.

Also not captured, but I would imagine this would continue to be the job of the bumpkin standing at the mailbox, even if not assigned any roles.

**Comment by vitt0** [comment]
I always tought that when this feature will be online we were going to have one bumpkin as main and the other ones as secondary without having the skill of them, reading into this proposal the first question that I have is why someone would want to level 1 bumpkin until 40 (for 27 skill points) when they can have for example 2 bumkins to lvl 17 (for 13 skill points each) with like 1/4~ of the total XP needed for 1 bumpking lvl 40?

I know that we don't only lvl up for the skill points, we also do it to unlock the expansions, but I still wonder how balancing this would work like. Even more that for example people are borrowing each other bumpkins to just expand the farm and sending them back to the original owners, making to have a high lvl bumpking even less important

**Reply by Kay1797** [reply]
valid points..

**Reply by Ogrekin** [reply]
lvl 40 bumpkin needed to unlock snow kingdom for example and maybe other islands that will come...
You need a high level bumpkin if you want to have access to everything 2 bumpkins at level 20 don't take you everywhere even if you borrow one stone heaven will be locked when you don't have a lvl 40 bumpkin in your island all the time and not just only when borrow one..  That my guess why you still need to have 1 high level bumpkin

**Comment by SarielWang93** [comment]
This proposal is very good, but in terms of skills, it should require the cooperation of a new skill tree, otherwise it may be abused. For example, the new skill tree points are only enough to upgrade one-sided skills.

**Reply by c-py** [reply]
💯 Will work with @TheDcol to make sure both proposals work hand in hand!

**Comment by future2back** [comment]
Reset skill idea/ multiple bumpkins are regressive, bumpkin wants to leave the farm, we can do more outside! How we going to visit the plaza with multiple bumpkins? How to participate in multiplayer quests/events with multiple bumpkins? How would that work with a guild system? How to keep multiple bumpkins fed? How to represent multiple bumpkins?

People gave up taking the EXP Skill, which would be very useful in this case thinking about the future. Do you think it's fair to these people? Sure that if this skill reset proposal came along with LE / Skill tree, the skill decision would be different for many users, now they will be harmed while others are boosted.

**Does a well-made skill tree not add value to the bumpkin?** 

How will the game handle metas?  Also it gets boring if everyone is the same.

**Reply by c-py** [reply]
> How we going to visit the plaza with multiple bumpkins? 

In the current design, this would be the role of the bumpkin standing next to the mailbox. If you wanted to visit with another bumpkin you could swap out that bumpkin.

> How to participate in multiplayer quests/events with multiple bumpkins?

This is a good point, quests would need to be considered. Right now the quest is per bumpkin. It could be left like this, OR, the quest could be limited to one per farm. @future2back Out of these two options, which one do you like better?

> People gave up taking the EXP Skill, which would be very useful in this case thinking about the future. Do you think it's fair to these people? Sure that if this skill reset proposal came along with LE / Skill tree, the skill decision would be different for many users, now they will be harmed while others are boosted.

> Does a well-made skill tree not add value to the bumpkin?

I think skill reset is a much wanted feature in game: https://github.com/sunflower-land/sunflower-land/discussions/2028
A well made skill tree definitely adds value to the game. I do think that skill tree changes and multiple bumpkins compliment each other, but features can coexist with or without each other.

> How will the game handle metas? Also it gets boring if everyone is the same.

A meta will always exist, with or without multiple bumpkins. Players try to min/max, it's in the nature of gaming.

**Reply by FelipeSVidal** [reply]
> Reset skill idea/ multiple bumpkins are regressive, bumpkin wants to leave the farm, we can do more outside! How we going to visit the plaza with multiple bumpkins? How to participate in multiplayer quests/events with multiple bumpkins? How would that work with a guild system? How to keep multiple bumpkins fed? How to represent multiple bumpkins?

On a farm there is a "farmer chief" that participe in exhibitions and contests, like Biggest pig contest, others bumpkins will are like the "family / employees" in land, each have especially abilities which help in farm grow. In this raciocine, first bumpkin (Mail Box) is chief.

**Comment by dragonhunter2017** [comment]
so, it means that we can't use green amulet and beetroot amulet together although they are on different bumkins?

**Reply by brian-man** [reply]
> so, it means that we can't use green amulet and beetroot amulet together although they are on different bumkins?

Kind of unfortunate.  My recollection is that the first comments about multi-bumpkin was that it gives you the ability to use the multiple amulets that some of us have accumulated.  Leveraging all the wearables is my primary reason for wanting multiple bumpkin support. :(

**Comment by spencerdezartsmith** [comment]
Nice job @c-py, I am excited for this. 

If I have one bumpkin with the Sunflower Amulet on and another with the Beetroot Amulet would I need to swap these bumpkins in and out of the Crop Role if I wanted to have them apply when planting Sunflowers and Beetroots respectively?

**Comment by Aquila199** [comment]
the amulet issue is important 

i should be able to use a sunflower with a green for example and i should not need to switch otherwise it kind of changes nothing from what we have now the different skills is cool and will be fun but i would like 3-4 bumpkins in different gear all effecting my farm without needing to switch to activate effects 

but we should not be able to stack as thats a game breaker imagine a person with multi GA ouch

**Comment by Xeko-92** [comment]
I would like for multiple wearables to take effect at once. For example, if I have the Green Amulet and the Carrot Amulet, I would like for those both to work. Otherwise, if there's a best piece of gear for a slot, all the others will be worthless if you have the good one.

**Reply by brian-man** [reply]
> I would like for multiple wearables to take effect at once. For example, if I have the Green Amulet and the Carrot Amulet, I would like for those both to work. Otherwise, if there's a best piece of gear for a slot, all the others will be worthless if you have the good one.

The original description of multi-bumpkin explicitly said that it would provide a solution for people to leverage at the same time multiple amulets.

Adam said in the stream today that the current direction would not allow this.

Makes me sad.  Using all my amulets at once was the only reason I wanted multiple bumpkins.

:(

**Comment by TheDcol** [comment]
As said by others, I think it would be better if all the bumpkins on the farm had their wearables active at once. I believe this was the main reason to get multiple bumpkins at the launch, and many times it was stated we could have all amulets active at once, eventually.

Skills can be adjusted to be locked at x level so that there is a reason to level up a bumpkin more than needed.

The roles aspect looks really interesting and immersive! Would the Golden Spatula wearable only work if it was assigned to a second bumpkin assigned to the kitchen?

I was brainstorming last few days about a system similar to what you proposed, would love to hear your thoughts:

Head Farmer - Assigned to Bumpkin A - SKILL TREE ACTIVE, ANY wearables work (sunflower amulet, shield)
Cooking Role - Assigned to bumpkin B - Cooking wearables ACTIVE (spatula, cake apron..)
Crop Role - Assigned to Bumpkin C - Crop wearables ACTIVE (Parsnip, green amulet.. )
Wood Role - Assigned to Bumpkin D - Wood wearables ACTIVE (incoming?)
Gathering Role - Empty

^ This way you have roles/specific purposes for each bumpkin whilst encouraging players to get more bumpkins and more wearables.
Only issue I see atm with what i mentioned above is the 4 crop amulets and only 2 roles for it

1 Solution would be to add a skill allowing a bumpkin to wear 2 amulets at once.. then the head farmer and crop role bumpkins will be able to wear all 4 amulets at once (just spitballing)

**Reply by Ogrekin** [reply]
This looks like a much better proposal than original proposal does, have a system where some of their nft become redundant
for some peoples and not for others is not a good system..
We can't just ignore the fact that the amulets exist and will be used together and I think there should be that possibility... 
Either that everyone can use amulets regardless of role and that there is the possibility of having several bumpkins or as suggested in this proposal. 
All bumpkins do not need to be connected to a skill tree, but that there is the possibility to use all wearables so none become redundant

**Reply by c-py** [reply]
Nice @TheDcol.

There is a handful of comments in support for multiple wearables (mainly amulets). I have also seen this sentiment in discord.

I think if we want to support players using multiple amulets, rather than designing a more complex system, we should remove "roles" from the multiple bumpkins feature. This reduces complexity of the feature and makes it easier for players to understand the impact of their multiple bumpkins. All bumpkins on the farm would have their boosts applied.

 @brian-man @TheDcol @andronicu3 @spencerdezartsmith @Xeko-92 @Aquila199 @dragonhunter2017 @vitt0 @SarielWang93 @future2back : How would you feel if we removed the "roles" feature from multiple bumpkins?

**Reply by Xeko-92** [reply]
Honestly, I didn't think Bumpkins would necessarily have specific roles when the discussion first started, I think it's a nice idea in theory, but in practice it just makes things more complicated. I think most people just want multiple bumpkins on their farm with all of their wearables taking effect.

Maybe we can choose one as the main bumpkin and have its talent tree work. I guess this might cause people to want to swap out bumpkins for different purposes though which isn't the best UX.

**Reply by Xeko-92** [reply]
If all the skill trees could take effect (but obviously not stack if 2 bumpkins have the same skill) while also introducing level requirements for certain skills or w/e, a way to balance and incentivize actually having higher level bumpkins, I think that would be best of both worlds.

**Reply by TheDcol** [reply]
> Nice @TheDcol.
> 
> There is a handful of comments in support for multiple wearables (mainly amulets). I have also seen this sentiment in discord.
> 
> I think if we want to support players using multiple amulets, rather than designing a more complex system, we should remove "roles" from the multiple bumpkins feature. This reduces complexity of the feature and makes it easier for players to understand the impact of their multiple bumpkins. All bumpkins on the farm would have their boosts applied.
> 
> @brian-man @TheDcol @andronicu3 @spencerdezartsmith @Xeko-92 @Aquila199 @dragonhunter2017 @vitt0 @SarielWang93 @future2back : How would you feel if we removed the "roles" feature from multiple bumpkins?

I think there are 2 main ways you can go:

1: You want to allow 3-4 bumpkins on the farm, ALL their wearables are active (but not stackable, 2 x green amulets would work as 1) and ALL their skills would be active too. <- Skills would need to be level locked to avoid abuse and we can discuss further.

2: You want to allow 3-4 bumpkins on the farm, ALL their wearables are active (but not stackable, 2 x green amulets would work as 1) BUT only 1 bumpkin has an 'active' skill tree. <- Roles would need to be in place with cooldown timers on switching (1, 3, 7+ days?) similar to what I posted above:
Head Farmer - Assigned to Bumpkin A - SKILL TREE ACTIVE, ANY wearables work (sunflower amulet, shield)
Cooking Role - Assigned to bumpkin B - Cooking wearables ACTIVE (spatula, cake apron..)
Crop Role - Assigned to Bumpkin C - Crop wearables ACTIVE (Parsnip, green amulet.. )
Wood Role - Assigned to Bumpkin D - Wood wearables ACTIVE (incoming?)
Gathering Role - Empty

I am.. indifferent. Option 1 is probably the simplest solution. Option 2 is more indepth and complex.

A third option is to go with Option 2 BUT add lite roles, if a bumpkin is on the farm and has full chef attire (golden spat, chef apron and chef hat) they get +5% xp from foods. A bumpkin with full warrior armour and sword/hammer gives +5% crop yield (example..). <- proper cooldowns would need to be in place as so players do not abuse.

Would also love some mechanic where bumpkins need to be fed for their wearables to work.. some sort of energy bar that needs to be above a certain threshold. Helps to avoid the snowball boost effect with a bit of food sink...

There are probably other ways to handle this but these are the ones I am thinking about

**Reply by ShiningMinions** [reply]
> I think if we want to support players using multiple amulets, rather than designing a more complex system, we should remove "roles" from the multiple bumpkins feature. This reduces complexity of the feature and makes it easier for players to understand the impact of their multiple bumpkins. All bumpkins on the farm would have their boosts applied.

Agreed. In fact, I wonder if an overhaul of the bumpkin skill-tree would actually be the answer here. You wouldn't "pre-select" a role, but rather, the skill-tree path you choose inadvertently becomes your "role". This assumes the skill-tree is redesigned in such a way that if you choose one path, you are actually sacrificing other paths.

@TheDcol I am wondering if your [Skilltree Revamp Proposal](https://github.com/sunflower-land/sunflower-land/discussions/2180) actually also strongly ties to this multiple bumpkins feature.

**Reply by TheDcol** [reply]
> @TheDcol I am wondering if your [Skilltree Revamp Proposal](https://github.com/sunflower-land/sunflower-land/discussions/2180) actually also strongly ties to this multiple bumpkins feature.

Potentially, the option is there. It can be done, depends on the direction of how they want to proceed.

Would have to sit down with a 'Heaps Normal' or two with @c-py

**Reply by TheDcol** [reply]
> You wouldn't "pre-select" a role, but rather, the skill-tree path you choose inadvertently becomes your "role". This assumes the skill-tree is redesigned in such a way that if you choose one path, you are actually sacrificing other paths.

More on this, it can definitely work BUT I would prefer adding level lock to the skills too, would be WAY too easy to unlock everything/most skills IMO without level requirements for x skills 

Bumpkin A: get to lvl 30, unlock most of crop tree
Bumpkin B: get to lvl 25, unlock most of fruit tree
Bumpkin C: get to level 25, unlock most of wood tree

I think more levels should be required to complete a skill tree if we are going down this route, level locking can help with this. Otherwise players will just stop levelling bumpkins up beyond a certain point.

Would also need to come up with a way to determine main bumpkin i think? for expansion/building requirements? or just pick highest level bumpkin?

A step further is making it so if we want a player to build a hen house, they need a bumpkin with minimum 1 skill unlocked in the animal tree

**Reply by vitt0** [reply]
> Nice @TheDcol.
> 
> There is a handful of comments in support for multiple wearables (mainly amulets). I have also seen this sentiment in discord.
> 
> I think if we want to support players using multiple amulets, rather than designing a more complex system, we should remove "roles" from the multiple bumpkins feature. This reduces complexity of the feature and makes it easier for players to understand the impact of their multiple bumpkins. All bumpkins on the farm would have their boosts applied.
> 
> @brian-man @TheDcol @andronicu3 @spencerdezartsmith @Xeko-92 @Aquila199 @dragonhunter2017 @vitt0 @SarielWang93 @future2back : How would you feel if we removed the "roles" feature from multiple bumpkins?

I think that I would rather have a simpler system without roles for multiple bumpkins, where there is only one main bumpkins that their skill are used and the others ones are secondary just to use extra wereables. If we want to add a little more complexity it better to work with something like the "Skill Tree REVAMP" proposal that this idea of bumpkins roles Imo 😄

**Reply by c-py** [reply]
Thanks @vitt0 , I agree, it seems there is enough support for removing roles and having multiple wearables proc. I think let's keep it simple and remove roles.

**Comment by Telknub** [comment]
I understand why bumpkin roles are not being added now, but I think this idea should be reconsidered after the skill rework.

Maybe the new skill system is balanced around having multiple bumpkins, in a way that a high level bumpkin can only complete 2 or 3 trees, making it necessary to have another bumpkin to  have more skills.

**Reply by c-py** [reply]
@Telknub yeah! The skill system will likely to be balanced.

There was also a mention of skills that don't fit into a role which helped to sway the change. A treasure based skill for example - it may become a bit hard to determine what role it should fit in, and more importantly, how to communicate this to the player.

Another example I can think is fruit trees, when you harvest, this would be a farmers job. When you cut them down, this is a lumberjack's job. I imagine it may limit new designs if we need to consider the roles in all upcoming features.

**Reply by Telknub** [reply]
The way I see the "roles" is that you would basically choose which bumpkin will use which skill tree. 

So the farmer role is related to the crops skill tree, the lumberjack role to the trees skill tree, a possible treasure role (maybe archeologist) to the treasure tree, etc... If a "role" doesnt fit in this case you could just simply use the name of the skill tree that bumpkin is using.

But for this to work the role system has to be worked with the skill rework at the same time, for a better balance.

**Reply by ShiningMinions** [reply]
> The way I see the "roles" is that you would basically choose which bumpkin will use which skill tree.

> So the farmer role is related to the crops skill tree, the lumberjack role to the trees skill tree, a possible treasure role (maybe archeologist) to the treasure tree, etc... If a "role" doesnt fit in this case you could just simply use the name of the skill tree that bumpkin is using.

> But for this to work the role system has to be worked with the skill rework at the same time, for a better balance.

 @Telknub @c-py  I think there could be some extreme fun if we can mix-and-match - it would just need some good balancing. For instance, you can choose to go "mediocre" between all skill trees (i.e. you won't be really good at one thing, but well-rounded in everything). Or alternatively, you can choose to go hardcore in only one skill tree - there would be amazing OP skills at the end of each skill tree, but in order to reach these you basically need to commit to only that skill tree because level/skillpoints requirements are so high.

This could lead to some really interesting gameplay,  as now "bumpkin building" is like it's own game. With this mechanic, I would be incentived to constantly mint new bumpkins and create new builds (and potentially sell those bumpkin builds on openSea).

**Reply by Telknub** [reply]
> > The way I see the "roles" is that you would basically choose which bumpkin will use which skill tree.
> 
> > So the farmer role is related to the crops skill tree, the lumberjack role to the trees skill tree, a possible treasure role (maybe archeologist) to the treasure tree, etc... If a "role" doesnt fit in this case you could just simply use the name of the skill tree that bumpkin is using.
> 
> > But for this to work the role system has to be worked with the skill rework at the same time, for a better balance.
> 
> @Telknub @c-py I think there could be some extreme fun if we can mix-and-match - it would just need some good balancing. For instance, you can choose to go "mediocre" between all skill trees (i.e. you won't be really good at one thing, but well-rounded in everything). Or alternatively, you can choose to go hardcore in only one skill tree - there would be amazing OP skills at the end of each skill tree, but in order to reach these you basically need to commit to only that skill tree because level/skillpoints requirements are so high.
> 
> This could lead to some really interesting gameplay, as now "bumpkin building" is like it's own game. With this mechanic, I would be incentived to constantly mint new bumpkins and create new builds (and potentially sell those bumpkin builds on openSea).

yeah, thats the idea I had. I think this would be really interesting.

**Comment by ImpactJoe** [comment]
This proposal seems quite preliminary as I was expecting more capabilities of having more than one bumpkin. Bumpkins hold tremendous weight due to their ability to work a farm and gain skill points to increase their farm capabilities. How about allowing players to mint land at level 1 with their additional bumpkin, just like one would with their main bumpkin. The tent could work as a portal to let’s say Bumpkin #2’s land and you can gain skill points and expand that land. This can even be a reduced version of the main bumpkin if that can potentially cause a problem down the road, but I think with such a robust set of developers you guys could pull this off.

**Reply by esp123** [reply]
Are you suggesting, multiple farms? The idea of multiple bumpkins is to have them in **1 farm** not create tons of farm which is illegal

**Reply by ImpactJoe** [reply]
Would it be possible to revisit the idea? Having multiple bumpkins in your wallet to have sort of sub-farms. They can be specialized for certain resources like having only plots for seeds.

**Comment by esp123** [comment]
Just a suggestion, what do you think if the 2nd bumpkin in tent will have his level and skill points linked to the main one.
Example:
Main bumpkin should be lvl 20 for 2nd bumpkin to lvl up to 10. This will limit players to place multiple low level bumpkins and force them to level up their main one to give the 2nd bumpkin more skills to get unlocked. 

Love also the idea of "roles" but maybe we can tied the idea of roles to a specific skill in the new "skill tree". What i mean is you want a "lumberjack bumpkin" you need to get a specific skill from the skill tree to be consider as one and with the high lvl 40 max lvl right now, will it be better for a single bumpkin to have multiple roles? Rather than calling them "roles", maybe a "title" suits more?

I also support the idea of multiple unique wearables active ingame. 

I would also love to see the tent changes depends on the "role" from the skill tree you get. That would be more unique 😍

**Comment by Galphath** [comment]
> Based on player feedback, roles have been removed from the Multiple Bumpkins proposal. The new design will allow wearables from multiple bumpkins to be used at one time

Maybe another tree, the main basic tree called foreman bumpkin, this one doesn't have paths but works by: 

 - Fulfilling the level requirements for buildings and expansions
 - Every x levels you could hire a new bumpkin
 - Boosts the other bumpkin's production, speed, feeding, ...

**Comment by minhtrongtgm** [comment]
How about this idea?
A bumpkin has 2 skill trees, temporary called "Primary" and "Secondary"
Primary is current skill tree.
Secondary may include some skills with small boost. It also bases on skill point as Primary but each skill need more points or any mechanism to make these skills to be only supporting skills
When a bumpkin is set to Primary, it's primary skill tree is applied, if it is set to Secondary, it's secondary skill tree is applied
Of course, allowing wearables from multiple bumpkins to be used at one time is very good idea. I think we can create more utility wearables 
"More farmers should more productivity" as real life  👩🏻‍🌾👩🏻‍🌾

**Comment by Blasta-alasta** [comment]
Not sure if it's been stated, but what about the idea of incorporating our bumpkins into upgrading our current buildings?

What I mean is this.... what if for a higher level Bumpkin you can unlock a feature to upgrade your Kitchen or Bakery, but this requires that you dedicate 2nd Bumpkin for this activity  (on top the usual sort of requirement for building), which essentially becomes an NPC attached to the upgraded building.

You could even have a Level Requirement for that 2nd Bumpkin which encourages advanced players to continue cooking more to level up more Bumpkins. Our 2nd Bumpkins wearables might be displayable as they work too which would be really cool. You would probably have to make existing buildings bigger to accomodate that 2nd Bumpkin though.

**Example:**

**_Upgrade Bakery_**

_Requirements:_

Main Bumpkin Level 50
500 wood
200 stone
5 gold
A (2nd) Level 10 Bumpkin
24 hours
(5 x 3 space)

What do you guys think?

**Comment by Fakir1Adam70** [comment]
2. Bumpkin should never be leveled. just use skills. 2nd bumpkin should base their level points on 1. bumpkin.
